Chummer/data/traditions.xml

```diff
@@ -941,6 +941,39 @@
         <spiritmanipulation>Spirit of Earth</spiritmanipulation>
       </spirits>
     </tradition>
+    <tradition>
+      <id>9d18592e-5f49-4992-babd-d1ac98c48f78</id>
+      <name>Shamanic [Traditionalist]</name>
+      <drain>{WIL} + {CHA}</drain>
+      <source>FA</source>
+      <page>74</page>
+      <addqualities>
+        <addquality forced="False">Mentor Spirit</addquality>
```

Comment:
They don't GET the quality, do they? I read that more as a requirement than a benefit.

Reply:
> They don't GET the quality, do they? I read that more as a requirement than a benefit.

The exact text in the book says:

> Must take the Mentor Spirit quality.

I read it the same as you. they do not seem to just get Mentor Spirit for free. Mentor Spirit looks more like it is a prerequisite.